.babelrc

```diff
@@ -1,11 +1,11 @@
 {
     "presets": ["es2015"],
     "plugins": [
+        "transform-class-properties",
         // this transforms async functions into generator functions, which
         // are then made to use the regenerator module by babel's
         // transform-regnerator plugin (which is enabled by es2015).
         "transform-async-to-bluebird",
-
```

Comment:
extreme nit: this blank line helps with legibility. Would be nice to keep it, and to insert one after line 4 for the comment block.

Reply:
Okay, will update!